package.json

```diff
@@ -5,7 +5,7 @@
   "main": "./dist/lib/index.js",
   "scripts": {
     "test": "npm run lint && npm run build && mocha dist/test/test-*.js",
-    "cover": "npm run build && istanbul cover _mocha -- dist/test/test-*.js",
+    "cover": "npm run build && istanbul cover ./node_modules/mocha/bin/_mocha -- dist/test/test-*.js",
```

Comment:
Unnecessary: npm already adds this to PATH.

Reply:
it doesn't work for me unless I add that path

Reply:
Hmmm, you're right: apparently this indeed doesn't work on Windows: https://www.npmjs.com/package/istanbul#usage-on-windows

So, I'm fine with leaving it like you proposed.